src/app/cli/src/graphql.ml

```diff
@@ -1006,6 +1024,20 @@ module Mutations = struct
         in
         public_key )
 
+  let reset_trust_status =
+    io_field "resetTrustStatus"
+      ~doc:"Reset trust status for a given IP address"
+      ~typ:(non_null Types.Payload.reset_trust_status)
+      ~args:Arg.[arg "input" ~typ:(non_null Types.Input.reset_trust_status)]
+      ~resolve:(fun {ctx= coda; _} () ip_address_input ->
+        let open Deferred.Result.Let_syntax in
+        let%bind ip_address =
+          Deferred.return
+          @@ Types.Arguments.ip_address ~name:"ip_address" ip_address_input
+        in
+        ignore (Coda_commands.reset_trust_status coda ip_address) ;
+        return ip_address )
```

Comment:
I think we shouldn't return the ip_addr, since it's always the same as the input.

Reply:
Yes, I had already fixed that in the code for the next PR. So I'll leave it in this one. Next PR will arrive when this one is merged.